spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/sql/TestPartitionedWritesAsSelect.java

```diff
@@ -18,34 +18,35 @@
  */
 package org.apache.iceberg.spark.sql;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 import java.util.List;
 import java.util.stream.IntStream;
 import org.apache.iceberg.spark.IcebergSpark;
-import org.apache.iceberg.spark.SparkTestBaseWithCatalog;
+import org.apache.iceberg.spark.TestBaseWithCatalog;
 import org.apache.spark.sql.types.DataTypes;
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestTemplate;
 
-public class TestPartitionedWritesAsSelect extends SparkTestBaseWithCatalog {
+public class TestPartitionedWritesAsSelect extends TestBaseWithCatalog {
 
-  private final String targetTable = tableName("target_table");
+  private final String targetTable = "testhadoop.default.target_table";
```

Comment:
why this change?

Reply:
aah this was unnecessary, thanks for catching this. Will revert.

Reply:
hmm reverting this change seems to cause more errors. Earlier, as we had constructors to initialize catalogName, we could simply use `tableName()`, but with parameters only, catalogName is not set, but passed as a `Parameter`. So we need to treat `targetTable` as a `Parameter` too.